pkgs/applications/networking/protonmail-bridge/default.nix

```diff
@@ -1,73 +1,43 @@
-{ stdenv, fetchurl, lib, qtbase, qtmultimedia, qtsvg, qtdeclarative, qttools, qtgraphicaleffects, qtquickcontrols2, full
-, libsecret, libGL, libpulseaudio, glib, wrapQtAppsHook, mkDerivation }:
+{ lib, buildGoModule, fetchFromGitHub, pkgconfig, libsecret }:
 
-let
-  version = "1.2.3-1";
-
-  description = ''
-    An application that runs on your computer in the background and seamlessly encrypts
-    and decrypts your mail as it enters and leaves your computer.
-
-    To work, gnome-keyring service must be enabled.
-  '';
-in mkDerivation {
+buildGoModule rec {
   pname = "protonmail-bridge";
-  inherit version;
+  version = "1.4.5";
 
-  src = fetchurl {
-    url = "https://protonmail.com/download/protonmail-bridge_${version}_amd64.deb";
-    sha256 = "032ggk9fvd19fbsqkzwzwh0hpyg8gpkrin71di7zsx6ias5innw1";
+  src = fetchFromGitHub {
+    owner = "ProtonMail";
+    repo = "proton-bridge";
+    rev = "br-${version}";
+    sha256 = "sha256-J7m1JcLrjUw0PIMoRWDXH1hax97/C7zq6Rj2o/SAaYw=";
```

Comment:
```suggestion
    sha256 = "J7m1JcLrjUw0PIMoRWDXH1hax97/C7zq6Rj2o/SAaYw=";
```

Reply:
On my side both works, and [some derivations](https://github.com/NixOS/nixpkgs/search?q=sha256-&type=code) use the same format as the one I'm using.

Reply:
And most of the others have the ``sha256-`` removed and those 7 packages should probably have it removed, too.